### DISCUSSION TITLE
Is there an ETA for 22.0.2?

### DISCUSSION THREAD

**Discussion by trixpan** [keycloak/keycloak#22632] [Q&A] [answered]
there are a number of bugs introduced in 22.0.0 that were not fixed by 22.0.2. Are there plans to release 22.0.2? 

Given the bugs continue to be raised, wouldn't it be worth releasing 22.0.2 and prepare for 22.0.3 as new things get found?

**Comment by jonkoops** [comment, marked as answer]
We do not give out exact timelines for releases, it will be done when we think it's in a good enough state to do a release. That said, we're expecting a release of 22.0.2 relatively soon.
